crates/sui-types/Cargo.toml

```diff
@@ -36,6 +36,7 @@ enum_dispatch.workspace = true
 eyre.workspace = true
 indexmap.workspace = true
 derivative.workspace = true
+tabled.workspace = true
```

Comment:
@amnn had a comment here about having really having the `tabled` crate in `sui-types`. I agree with him. Despite the crate being pretty useful, it's still under development and not super mature, so might want to consider if we can implement these pretty stuff in a different crate.